### PR TITLE
tests: remove deprecated hamcrest.contains

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

**Review by xxbillouxx — CHANGES_REQUESTED** [review]
Reviewed. gj

File: integration_tests/suite/helpers/base.py [hunk 1]
```diff
@@ -385,5 +385,5 @@ def assert_sorted(action, order, expected):
     assert_that(
```
[the remark this answers is not in the document]
Comment (sduthil) [reply integration_tests/suite/helpers/base.py:385]: `action` is a function like the others `fn` that runs an HTTP request. I prefer to change the other `fn` with `action` which is a bit more explicit.

[the remark this answers is not in the document]
Comment (xxbillouxx) [reply integration_tests/suite/helpers/base.py:385]: `action`, without any context, has no meaning. So as a developer I prefer `fn`.

And as you said `action is a function` :-)

**Comment by xxbillouxx** [comment]
recheck

**Review by xxbillouxx — APPROVED** [review]
gj

**Pull request merged**